src/System.Private.Uri/src/System/UriExt.cs

```diff
@@ -499,29 +499,46 @@ internal unsafe bool InternalIsWellFormedOriginalString()
         public static string UnescapeDataString(string stringToUnescape)
         {
             if ((object)stringToUnescape == null)
+            {
```

Comment:
Yes I was aware of single line blocks with no braces was allowed in our guidelines, I just added them because in other PRs I've done I have had the experience where the owners asked for braces on those blocks :( so I didn't know which one was the preferred style, but will get rid of them in this case :)